tex/chapters/survey/main.tex

```diff
@@ -1,6 +1,71 @@
 \chapter{Data collection}
 
 \section{Background on survey design}
+
+%cite: https://books.google.co.uk/books?id=mSRTDwAAQBAJ&lpg=PP1&ots=nynKGurXVG&dq=how%20to%20write%20a%20survey&lr&pg=PP1#v=onepage&q&f=false
+%cite: https://www.idsurvey.com/en/advantages-and-disadvantages-of-phone-survey/
+
+A survey is a method of data collection with the purpose of obtaining quantitative information regarding opinions and 
+experiences of the respondents. Surveys could be used to investigate, for example, a product for which a company would like 
+consumer feedback, or how a disease impacts patients in their daily lives. This form of investigation is useful for gathering a
+large amount of data that can be easily quantified and analysed using statistical methods and tests. 
+
+Survey methodology involves asking a sample of the target population a series of standardised questions, which may be given in 
+the form of written questionaires or spoken interviews. Before the type of survey is decided, it is important to consider the 
+aims and objectives of the research. 
+
+For example, phone calls and other forms of interview-based survey allow the interviewer to form a personal connection with 
+the survey participant, which can be especially helpful for a company's image if the interviewer is particularly professional 
+or charismatic. This type of survey also provides an instant response, which is beneficial if there is only a short time frame 
+available in which to gather information. Furthermore, any error as a result of participants misinterpreting questions can be 
+negated due to the interviewer having the ability to immediately clarify on any misunderstandings. 
+
+However, there are also shortfalls to an interview-based survey method.
+
+
+
+
+% About Questionaires
+A questionaire is composed of two types of questions; closed-ended questions, where participants select answers from a list,
+and open-ended questions, in which the participants are able to write their own answers. 
+
+
+
 \section{Specific goals of survey for this chapter}
+
+
 \section{The survey}
+
+\textbf{Cohorts}
+\begin{itemize}
+    \item Students
+    \item People who work in data visualisation and/or statistical programming
+    \item General population
+\end{itemize}
+
+\textbf{Information about participants}
+
+\begin{itemize}
+    \item Please select your age category. (Checkbox: Under 18, 18-25, 26-35, 36-45, Above 45)
+    
+    \item If you are a university student or past university graduate please specify your area of study. (maybe drop down box: Science, Technology, Engineering, Mathematics, Arts, Social Sciences, Humanities, Business, Other, N/A)
+    
+    \item How would you rate your spatial awareness skills? (Scale/slider: Poor, Okay, Good, Very Good)
+    
+    \item How would you rate your observation skills? (Scale/slider: Poor, Okay, Good, Very Good)
```

Comment:
Same comment about awareness skills.

I wonder if both of these questions should be written in the style.

> How strongly do you agree with the following statement:

> "I consider myself to have above average spatial awareness"

But we'd need to know why it is that people use that form for questions.